Language Tools + Time and Complexity Assignment/Duplicate_in_array.cpp

```diff
@@ -15,17 +15,15 @@ Sample Output:
 
 // arr - input array
 // size - size of array
-//ritikagarwal231
+//RohanKumar01
 #include<algorithm>
 int findUnique(int *arr, int n) {
```

Comment:
### Can you update the function name in your pull request?
<img width="842" alt="Screenshot 2021-08-07 at 8 03 47 PM" src="https://user-images.githubusercontent.com/47470038/128603672-cedac6db-c4c3-4c36-8b9c-5d377af3b3cf.png">

Reply:
yeah sure.